doc/releases/release-notes-2.1.rst

```diff
@@ -96,6 +96,11 @@ Build and Infrastructure
   Use new functions that utilize eDTS info such as dt_node_reg_addr.
   See :zephyr_file:`scripts/kconfig/kconfigfunctions.py` for details.
 
+* Deprecated direct use of the `DT_` Kconfig symbols from the generated
+  `generated_dts_board.conf`.  We should now utilize either functions from
+  `scripts/kconfig/kconfigfunctions.py` for kconfig or functions in
```

Comment:
Might be nice to link to https://docs.zephyrproject.org/latest/guides/kconfig/index.html#kconfig-functions here.

Reply:
Example of how?

Reply:
Thinking something like

"We should now either use the functions from :zephyr_file:`scripts/kconfig/kconfigfunctions.py` (see <url>), or..."